desktop/docs/development.md

```diff
@@ -15,43 +15,34 @@ We use Electron's [IPC](https://github.com/atom/electron/blob/master/docs/api/ip
+- _(main)_ A [preload script](../public_desktop/preload.js) is added to the `BrowserView` configuration. Electron executes this preload script before all other scripts when a page is loaded in the view, and injects Inter-Process Communication ("IPC") channels into the Renderer (browser) process that facilitate things like fetching authentication credentials and keyboard/menu shortcuts to specific locations within Calypso. Communication to/from Calypso is defined in the Calypso's [desktop listeners](../../client/lib/desktop-listeners/index.js) file.
+  - _(renderer)_ Calypso's default [configuration](../../packages/calypso-config/src/index.ts) is overriden by the Desktop application at runtime via the presence of the `window.electron` object in order to affect the behavior of the WordPress.com webapp in select circumstances.
+- _(main)_ After the `BrowserWindow` has been created, various [window handlers](../app/window-handlers/README.md) are loaded that add behaviors to the window and view instances (spellchecking, navigation, notifications, etc).
+- _(renderer)_ In addition to the preload script, the `desktop/public_desktop` folder also contains other assets that are loaded into the Renderer process independent of Calypso, such as HTML and CSS for the application's Preferences and About panes. For convenience, the application's icons are located here as well.
 
 Phew!
 
 ## How do I change the main app?
 
-All app code is contained in the `desktop` and `client/desktop` directories. Any changes you make there require a rebuild and restart of the app.
+All app code is contained in the `desktop` directory, and desktop overrides are implemented in Calypso as-needed (using the `config.isEnabled( 'flag' )` mentioned in the prior section). Changes you make to the Desktop's application Node process (main) logic require a rebuild and restart of the app.
 
 - [Config](../desktop-config/README.md) - app configuration values
-- [Libraries](../../client/desktop/lib/README.md) - details of the local libraries used
-- [App Handlers](../../client/desktop/app-handlers/README.md) - handlers that run before the main window is created
-- [Window Handlers](../../client/desktop/window-handlers/README.md) - handlers that run after the main window is created
+- [Libraries](../app/lib/README.md) - details of the local libraries used
+- [App Handlers](../app/app-handlers/README.md) - handlers that run before the main window is created
+- [Window Handlers](../app/window-handlers/README.md) - handlers that run after the main window is created
+
+The application can also be run in a development mode that uses the localhost instance of Calypso, instead of loading the production WordPress.com webapp. Refer to the main README for details.
```

Comment:
We could likely add a link to the main README here to make it easy to get to and match the others above.